.github/workflows/verify-modernization-ui.yaml

```diff
@@ -0,0 +1,34 @@
+# This workflow is used to verify the code quality by running tests
+name: Verify Modernization UI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "apps/modernization-ui/**"
+
+env:
+  passed_github_token: ${{secrets.PASSED_GITHUB_TOKEN}}
+
+jobs:
+  pipeline:
+    name: Build and verify
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
```

Comment:
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/4cd63058-3341-49c8-b91e-5f619e799b6b)

Do we want all?

Reply:
Good question, I copied this from the sonar pipeline

Reply:
Removing with fetch-depth is typical. You just want the latest commit